src/convertSourceToRaw.m

```diff
@@ -1,16 +1,19 @@
-function convertSourceToRaw(cfg)
+function convertSourceToRaw(varargin)
     %
     % Function attempts to convert a source dataset created with CPP_BIDS into a valid
     % BIDS data set.
     %
     %
     % USAGE::
     %
-    %   convertSourceToRaw(cfg)
+    %   convertSourceToRaw(cfg, 'filter', filter)
```

Comment:
Should now be possible to use a typical bids.query fitler to only convert a subset of files (only those from `beh` or from a given subject)